CRMApi/AWS/DynamoDB/Linq/InvalidQueryException.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+namespace CRMApi.AWS.DynamoDB.Linq {
+    class InvalidQueryException : Exception {
+        private string message;
+
+        public InvalidQueryException(string message) {
+            this.message = message + " ";
```

Comment:
Is the extra `+ " "` needed?  I thought the base `Exception` class already had a ctor that accepted a message.